README.md

```diff
@@ -46,101 +46,50 @@ This repository contains the ape-backend codebase which serves the Coordinape AP
 
 This is a project written in PHP using the [Laravel](https://laravel.com/) PHP framework. The database is mysql. Docker is used to setup the environment.
```

Comment:
"The database is mysql." s/mysql/postgresql

Reply:
```suggestion
This is a project written in PHP using the [Laravel](https://laravel.com/) PHP framework. The database is postgresql. Docker is used to setup the environment.
```

i recommend using the "insert a suggestion" feature for this sort of thing

<img width="366" alt="Disable_Migrations_on_Laravel_Start_by_exrhizo_·_Pull_Request__74_·_coordinape_coordinape-backend" src="https://user-images.githubusercontent.com/193667/150893887-f82a4100-99a6-43c1-889f-450bc2cda802.png">